dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java

```diff
@@ -36,7 +36,7 @@ public static MappingCacheManager getInstance(ScopeModel scopeModel) {
         return scopeModel.getBeanFactory().getOrRegisterBean(MappingCacheManager.class);
     }
 
-    public MappingCacheManager(String name, ScheduledExecutorService executorService) {
+    public MappingCacheManager(boolean fileCache, String name, ScheduledExecutorService executorService) {
```

Comment:
rename `fileCache` field to `enableFileCached`

Reply:
Sorry, I have made a spelling mistake. The proper filed name is `enableFileCache` which means whether enable file cache or not.

Reply:
OK